TODOs/Classes/Models/Setting.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Setting.swift
+//  TODOs
+//
+//  Created by Kevin Johnson on 9/10/20.
+//  Copyright © 2020 Kevin Johnson. All rights reserved.
+//
+
+import Foundation
+
+struct Setting: Codable {
+    enum Frequency: Int, Codable, CaseIterable {
+        case sundays = 0
+        case mondays
+        case tuesdays
+        case wednesdays
+        case thursdays
+        case fridays
+        case saturdays
+        case weekends = 100
+        case weekdays
+        case everyday
+
+        var description: String {
+            switch self {
+            case .sundays,
+                 .mondays,
+                 .tuesdays,
+                 .wednesdays,
+                 .thursdays,
+                 .fridays,
+                 .saturdays:
+                return Calendar.current.shortWeekdaySymbols[self.rawValue]
+            case .weekends:
+                return NSLocalizedString("Weekends", comment: "")
+            case .weekdays:
+                return NSLocalizedString("Weekdays", comment: "")
+            case .everyday:
+                return NSLocalizedString("Everday", comment: "")
+            }
+        }
+    }
+    var name: String
+    var frequency: Frequency
```

Comment:
changing rn won't modify already modified todos, but see how this feels when using